autobuild/appdeps.py

```diff
@@ -0,0 +1,144 @@
+import argparse
```

Comment:
I made it so that it only copies dependencies in "-L" directories (to avoid copying system dependencies in /usr/lib).
Try something like this
```
python3 ./appdeps.py --outdir ./outd -L<vcpkg-installed-dir>/x64-linux-synfig/lib/ ~/projects/synfig/synfig-master-clean/build/output/Release/bin/synfig
```